README.md

```diff
@@ -2,7 +2,9 @@
 Gluon Ignite allows developers to use popular dependency injection frameworks in their JavaFX applications, 
 including inside their FXML controllers. Gluon Ignite creates a common abstraction over several popular 
 dependency injection frameworks (currently _[Guice](https://github.com/google/guice), [Spring](https://spring.io/)_, 
-and _[Dagger](https://square.github.io/dagger/)_, but we plan at add more as the demand becomes obvious). 
+, and _[Dagger](https://square.github.io/dagger/)_). 
```

Comment:
this has `, ,` now

Reply:
Moved to `1.1.0-SNAPSHOT` and updated the docs accordingly.